client/my-sites/plan-features-comparison/header.jsx

```diff
@@ -58,6 +58,10 @@ export class PlanFeaturesComparisonHeader extends Component {
 			isMonthlyPlan,
 		} = this.props;
 
+		if ( rawPrice === 0 ) {
+			return translate( 'Free forever' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'free forever' )` *ES Score: 12*